mackup/applications/bc.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = bc
+
+[configuration_files]
+.bcrc
```

Comment:
Can you end the file with a newline char?

Reply:
should be good now.